sfdx/README.md

```diff
@@ -21,198 +21,232 @@ A plugin for the Salesforce CLI to create mobile applications to interface with
 ## Help
 ```
 -> sfdx mobilesdk --help
```

Comment:
This readme is generated using `sfdx/generate_readme.sh`.
It looks like the help output has changed in the latest version of `sfdx`.